src/common/perf_counters.h

```diff
@@ -179,6 +179,10 @@ class PerfCounters
     m_name = s;
   }
 
+  void set_suppress_nicks(bool b) {
```

Comment:
Why would one code this rather than just setting the nick to ""?  If it were dynamic, that's one thing, but given you have to change the C++ code either way...?